### PR TITLE
Added CLI version of real-time alerting solution

### PR DESCRIPTION
**Pull request by diawarr**
- CLI version of the real-time alerting solution instructions
- Some small changes to the console version (moved steps on getting Kafka service details to the section for sample data, as they're only needed there)

### CODE REVIEW

File: docs/products/flink/howto/real_time_alerting_solution_cli.rst [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+   ::
+
+      avn service get kafka-demo --format '{service_uri_params}'
+
+#. Run the following Python command to create the sample records using the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ tool:
+
+   ::
+
+      python3 python-fake-data-producer-for-apache-kafka/metricproducer.py \
```
Comment (ftisiot) [inline docs/products/flink/howto/real_time_alerting_solution_cli.rst:121]: Can we replace this with the docker version https://github.com/aiven/fake-data-producer-for-apache-kafka-docker

File: docs/products/flink/howto/real_time_alerting_solution_cli.rst [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+    graph LR;
+
+        id1(Kafka)-- metrics stream -->id3(Flink);
+        id2(PostgreSQL)-- threshold data -->id3;
+        id3-. filtered data .->id4(Kafka);
+        id3-. filtered data .->id5(PostgreSQL);
+
+The article includes the steps that you need when using the `Aiven CLI <https://github.com/aiven/aiven-client>`_ along with a few different samples of how you can set thresholds for alerts. For connecting to your PostgreSQL service, this example uses the Aiven CLI calling `psql <https://www.postgresql.org/docs/current/app-psql.html>`_, but you can also use other tools if you prefer.
+
+In addition, the instructions show you how to use a separate Python tool, `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_, to create sample records for your Apache Kafka topic that provides the streamed data.
```
Comment (ftisiot) [inline docs/products/flink/howto/real_time_alerting_solution_cli.rst:22]: replace the link to point to the docker version https://github.com/aiven/fake-data-producer-for-apache-kafka-docker

File: docs/products/flink/howto/real_time_alerting_solution_cli.rst [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+      {"hostname": "dopey", "cpu": "cpu4", "usage": 98.3335306302198, "occurred_at": 1633956789277}
+      {"hostname": "sleepy", "cpu": "cpu2", "usage": 87.28240549074823, "occurred_at": 1633956783483}
+      {"hostname": "sleepy", "cpu": "cpu1", "usage": 85.3384018012967, "occurred_at": 1633956788484}
+      {"hostname": "sneezy", "cpu": "cpu1", "usage": 89.11518629380006, "occurred_at": 1633956781891}
+      {"hostname": "sneezy", "cpu": "cpu2", "usage": 89.69951046388306, "occurred_at": 1633956788294}
+
+
+Create a pipeline for basic filtering
+-------------------------------------
+
```
Comment (ftisiot) [inline docs/products/flink/howto/real_time_alerting_solution_cli.rst:148]: Can we add a quick diagram stating what we do with SOURCE->FLINK_JOB_NAME->TARGET?

**Review by ftisiot — CHANGES_REQUESTED** [review]
A couple of minor changes requested (the docker reference is valid for both console and cli) and also
* should we have a little diagram showing SOURCE->FLINK_JOB->TARGET at the beginning of each session
* As suggested by Soumya, should we cross reference the content (e.g. Watermarks, Connection types, Data Formats)

File: docs/products/flink/howto/real_time_alerting_solution.rst [hunk 4]
```diff
@@ -56,34 +50,41 @@ Set up Aiven services
 Set up sample data
 ------------------
 
-Run the following Python command to create the sample records using the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ tool:
+Before you start, clone the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ Git repository to your computer and then run ``pip install -r requirements.txt`` in your local copy to make sure that the necessary dependencies are installed.
```
[the remark this answers is not in the document]
Comment (diawarr) [reply docs/products/flink/howto/real_time_alerting_solution.rst]: Replaced the steps with instructions for the dockerized tool, so this change is no longer needed ;)

File: docs/products/flink/howto/real_time_alerting_solution_cli.rst (hunk 3 above)
Comment (diawarr) [reply docs/products/flink/howto/real_time_alerting_solution_cli.rst:150]: Yep, I've added those to both console and CLI versions of the instructions ;)

**Comment by diawarr** [comment]
Most of those changes are now in place, but:
- The link in the CLI instructions for creating the authentication token currently points to the production URL, as the topic is not yet in the target branch. This can be updated to a relative link once the content is rebased.
- There are now cross-reference links to the concepts on windows and event time, but nothing yet for connectors or data formats. These can be added once the relevant concepts are available.

**Review by ftisiot — APPROVED** [review]
Good content!

**Pull request merged**